osu.Game/Overlays/Comments/ShowChildrenButton.cs

```diff
@@ -3,32 +3,36 @@
 
 using osu.Framework.Graphics;
 using osu.Game.Graphics.Containers;
-using osu.Framework.Input.Events;
 using osu.Framework.Bindables;
+using osu.Game.Graphics;
+using osuTK.Graphics;
+using System.Collections.Generic;
+using osu.Game.Online.API.Requests.Responses;
 
 namespace osu.Game.Overlays.Comments
 {
     public abstract class ShowChildrenButton : OsuHoverContainer
     {
         public readonly BindableBool Expanded = new BindableBool(true);
+        public readonly Bindable<List<Comment>> ChildComments = new Bindable<List<Comment>>();
 
         protected ShowChildrenButton()
         {
             AutoSizeAxes = Axes.Both;
+            Action = () => Expanded.Value = !Expanded.Value;
+
+            IdleColour = OsuColour.Gray(0.7f);
+            HoverColour = Color4.White;
         }
 
         protected override void LoadComplete()
         {
             Expanded.BindValueChanged(OnExpandedChanged, true);
+            ChildComments.BindValueChanged(OnChildrenChanged, true);
```

Comment:
Same here, this is also not the proper way to do this (mutating your own visual state). It's the responsibility of the user of this button to do so.